### ISSUE TITLE
How to reference linearization views?

### ISSUE THREAD

**Issue by samsontu** [who-icatx/whofic-models#23]
Currently, in the linearization specifications of an ICD entities, we reference the linearizationViews using IRIs (e.g., "linearizationView": "http://id.who.int/icd/release/11/mms"

The model for the LinearizationView also as a string Id: 
  LinearizationView:
    is_a: ContentModelEntity
    description: >-
      The schema of a linearization
    slots:
      - Id
      - Description
      - rootId
      - LinearizationMode
      - CoreLinId
      - sortingCode
      - defaultIsIncluded
      - defaultIsGrouping
      - defaultIsAuxiliaryAxisChild

Is there a good reason to use an IRI versus a string id (e.g., "MMS")?

**Comment by tudorache** [comment]
No, we can use either. I would use an IRI to keep in line with the OWL id conventions.

In the [linearizationViewDefinition](https://drive.google.com/file/d/1E03IKkg9udcOFTZmV8w1oKHGldj9XBJv/view?usp=drive_link) file, we have both a String id ("MMS") and the IRI ("http://id.who.int/icd/release/11/mms") that we can use as ID.
